nautobot/extras/forms.py

```diff
@@ -660,7 +728,7 @@ class Meta:
 class ExportTemplateFilterForm(BootstrapMixin, forms.Form):
     model = ExportTemplate
     q = forms.CharField(required=False, label="Search")
-    content_type = forms.ModelChoiceField(
+    content_type = CSVContentTypeField(
```

Comment:
This field change for `ExportTemplateFilterForm` was required because the filtering *was not actually even working for this field*.